src/backend/demo/management/commands/create_demo.py

```diff
@@ -143,6 +143,8 @@ def create_demo(stdout):
             queue.push(
                 models.Team(
                     name=f"Team {i:d}",
+                    # slug should be automatic but bulk_create doesn't use save
+                    slug=f"Team-{i:d}",
```

Comment:
while you're at it, make it a real slug :stuck_out_tongue: 
```suggestion
                    slug=f"team-{i:d}",
```